networking/p2p/src/test/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/rpc/encodings/LengthPrefixedEncodingTest.java

```diff
@@ -107,8 +108,30 @@ public void shouldThrowRpcExceptionIfMessageLengthPrefixIsMoreThanThreeBytes() {
         .isEqualTo(RpcException.CHUNK_TOO_LONG_ERROR);
   }
 
+  @Test
```

Comment:
Should we add a test for `String` encoding as well?

Reply:
Yes good spot.